apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/ILogQueue.cs

```diff
@@ -12,44 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
 using System.Collections.Generic;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Google.Cloud.Logging.Log4Net
 {
     internal interface ILogQueue
     {
         /// <summary>
-        /// Get the next ID to use for sequential log entry numbering.
+        /// Get the latest ID used in a possible previous execution.
         /// </summary>
-        /// <returns></returns>
-        Task<long> GetNextIdAsync();
+        /// <returns>The latest ID froma previous exeuction, or <c>null</c> if not relevant.</returns>
+        Task<long?> GetPreviousExecutionIdAsync();
 
         /// <summary>
         /// Enqueue the given log entries
         /// </summary>
         /// <param name="logEntries"></param>
         /// <returns>The daterange lost, or null if nothing lost.</returns>
-        DateTimeRange EnqueueAsync(IEnumerable<LogEntryExtra> logEntries);
+        DateTimeRange Enqueue(IEnumerable<LogEntryExtra> logEntries);
```

Comment:
daterange => date range? (In doc above this line) And maybe add a couple of words, e.g.

```csharp
/// <returns>The date range of lost entries, or null if nothing was lost.</returns>
```

It might be worth adding a remarks section to explain about lost entries. (And add full stop in the summary?)